fic/resource_eri_router_paired_to_gcp_connection_v1_test.go

```diff
@@ -0,0 +1,211 @@
+package fic
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+
+	"github.com/nttcom/go-fic"
+
+	connections "github.com/nttcom/go-fic/fic/eri/v1/router_paired_to_gcp_connections"
+
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccPairedRouterToGCPConnection_basic(t *testing.T) {
+	var connection connections.Connection
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "fic_eri_router_paired_to_gcp_connection_v1.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:      func() { testAccPreCheckGCPConnection(t) },
+		Providers:     testAccProviders,
+		CheckDestroy:  testAccCheckPairedRouterToGCPConnectionDestroy,
+		IDRefreshName: resourceName,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccPairedRouterToGCPConnectionConfig(rName, "10M", "noRoute", "privateRoute", 10),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckPairedRouterToGCPConnectionExists(resourceName, &connection),
+					resource.TestCheckResourceAttr("fic_eri_router_paired_to_gcp_connection_v1.test", "name", rName),
```

Comment:
refreshに関する質問を噛み砕いた結果、以下のような変更を加えても問題はなくコードがスッキリすると考えたのですが、いかがでしょうか
```suggestion
					testAccCheckPairedRouterToGCPConnectionExists(resourceName, &connection),
					resource.TestCheckResourceAttr(resourceName, "name", rName),
```

Reply:
ありがとうございます。修正しました。